core/src/main/java/org/apache/calcite/sql/dialect/SparkSqlDialect.java

```diff
@@ -389,6 +409,18 @@ private void unparseIntervalOperandCall(
     writer.print(")");
   }
 
+  private void unparseIntervalOperandCallWithBinaryOperator(
+      SqlCall call, SqlWriter writer, int leftPrec, int rightPrec) {
+    call.operand(0).unparse(writer, leftPrec, rightPrec);
+    if (call.getKind() == SqlKind.MINUS) {
+      writer.sep("-");
+    } else {
+      writer.sep("+");
+    }
+    SqlNode intervalValue = modifySqlNode(writer, call.operand(1));
+    writer.print(intervalValue.toString().replace("`", ""));
```

Comment:
Please use unparse method for intervalValue

Reply:
done